File: sample/sample.c
#include <stdio.h>

int main(void)
{
    int i, j, k, turn, finger[2][2][4];
    while(1) {
        scanf("%d", &turn);
        for(i = 0; i < 2; i++)
            for(j = 0; j < 2; j++)
                for(k = 0; k < 4; k++)
                    scanf("%d", &finger[i][j][k]);
        printf("%d %d %d\n", rand() % 2, rand() % 2, rand() % 4);
        fflush(stdout);
    }
    return 0;
}

File: sample/sample.py
#!/usr/bin/env python
import sys
import random

while True:
	turn = int(raw_input())
	finger = [[], []]
	for i in range(0, 2):
		for j in range(0, 2):
			finger[i].append([int(k) for k in raw_input().split()])

	print '%d %d %d' % (random.randint(0, 1), random.randint(0, 1), random.randint(-3, 3))
	sys.stdout.flush()
